packages/component-library/components/Button/components/GenericButton.tsx

```diff
@@ -36,7 +36,7 @@ type LabelProps = {
 
 type Analytics = {
   eventName: string
-  properties: object
+  properties: Record<string, unknown>
```

Comment:
I'm unsure whether this needs to remain `object` for the purposes of consistency. Can anyone more familiar weigh in on this?